Reduce amount of cmake log from building & downloading libArrow (#4103)
rerun-io/rerun: rerun_cpp/CMakeLists.txt

        PREFIX ${ARROW_DOWNLOAD_PATH}
        GIT_REPOSITORY https://github.com/apache/arrow.git
        GIT_TAG apache-arrow-10.0.1
        GIT_SHALLOW ON

        # Git progress sounds like a nice idea but is in practive very spammy.

>         # GIT_PROGRESS ON

Reviewer: Why not set it to OFF then?

Author: it's default off, but yeah why not

>         # LOG_X ON means that the output of the command will *not* be printed to the console,
>         # but instead be directed to a file (it logs where the file goes).

Reviewer: ```suggestion
        # LOG_X ON means that the output of the command will
        # be logged to a file _instead_ of printed to the console.
```

🤷
